SU-SWS/stanford_courses
# HSDO-874 removed if statement around the hook_views_pre_build logic

# READY FOR REVIEW

# Summary
- Changed the hook_views_pre_build() to work on any view, not just the views in the feature. 

# Needed By (Date)
- 7/5

# Urgency
- Low urgency

# Steps to Test

1. checkout branch, revert all courses features
2. clone the view that is on /courses page. then disable the original view.
3. view /courses page
4. if courses display, filter til you get no courses.
5. Verify the 'Search courses' button go to an actual page within the site.
